app/views/trips/_trip.html.erb

```diff
@@ -1,7 +1,7 @@
 <div id="<%= dom_id trip %>">
   <p>
     <strong>User:</strong>
-    <%= trip.user_id %>
+    <%= User.fullname(trip.user).pluck(:name, :surname) %>
```

Comment:
`current_user.fullname`

Reply:
We can't use `current_user` because it will change all trips to have our name instead of the trip's owner name